### ISSUE TITLE
@winglibs/cognito file not found: "./aws-utils.ts"

### ISSUE THREAD

**Issue by ekeren** [winglang/wing#6086]
### I tried this:

```sh
npm i @winglibs/cognito 
```

```wing
bring cognito;
```

```
wing run 
```

### This happened:

```
An error has occurred:
Failed to compile.

error: File not found: "./aws-utils.ts"
    --> node_modules/@winglibs/cognito/platform/tfaws.w:133:3
    |
133 |   extern "./aws-utils.ts" static pub inflight _signUp(clientId: str, email: str, password: str): void;
    |   ^^^^^^^^^^^^^^^^^^^^^^^ File not found: "./aws-utils.ts"


error: File not found: "./aws-utils.ts"
    --> node_modules/@winglibs/cognito/platform/tfaws.w:134:3
    |
134 |   extern "./aws-utils.ts" static pub inflight _adminConfirmUser(poolId: str, email: str): void;
    |   ^^^^^^^^^^^^^^^^^^^^^^^ File not found: "./aws-utils.ts"


error: File not found: "./aws-utils.ts"
    --> node_modules/@winglibs/cognito/platform/tfaws.w:135:3
    |
135 |   extern "./aws-utils.ts" static pub inflight _initiateAuth(clientId: str, email: str, password: str): str;
    |   ^^^^^^^^^^^^^^^^^^^^^^^ File not found: "./aws-utils.ts"


error: Symbol "Cognito" has multiple definitions in "/private/tmp/demo-simtools/node_modules/@winglibs/cognito/platform"
  --> main.w:3:1
  |
3 | bring cognito;
  | ^^^^^^^^^^^^^^ Symbol "Cognito" has multiple definitions in "/private/tmp/demo-simtools/node_modules/@winglibs/cognito/platform"

```

### I expected this:

to work

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by Chriscbr** [comment]
Oof... yeah the compiler should be catching these errors before we publish the library. I'll try to fix and open a bug for the compiler (also need one for the ts issue)

**Comment by ekeren** [comment]
What is the root cause for this?

**Comment by ekeren** [comment]
Oh, I see 
@eladcon has 2 pub class under the same directory with name `Cognito`

**Comment by Chriscbr** [comment]
Fixed in cognito 0.0.4

I've opened this issue for the symbol conflict error (compiler bug) https://github.com/winglang/wing/issues/6087
